docs/src/rules/sort-imports.md

```diff
@@ -278,6 +293,17 @@ import {b, a, c} from 'foo.js'
 
 :::
 
+Examples of **correct** code for this rule with the default `{ "ignoreMemberSort": false }` option:
+
+::: correct
+
+```js
+/*eslint sort-imports: ["error", { "ignoreMemberSort": false }]*/
+import {a, b, c} from 'foo.js'
```

Comment:
```suggestion
import {a, b, c} from 'foo.js';
```

and an `incorrect` section can also be added for { "ignoreMemberSort": true } with following example:
```js
import b from 'foo.js';
import a from 'bar.js';
```